homeassistant/components/lightwave/sensor.py

```diff
@@ -0,0 +1,67 @@
+"""Support for LightwaveRF TRV - Associated Battery."""
+import logging
+
+from homeassistant.const import CONF_NAME, DEVICE_CLASS_BATTERY
+from homeassistant.helpers.entity import Entity
+
+from . import CONF_SERIAL, LIGHTWAVE_LINK
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Find and return battery."""
+    if not discovery_info:
+        return
+
+    batt = []
+
+    lwlink = hass.data[LIGHTWAVE_LINK]
+
+    for dummy, device_config in discovery_info.items():
```

Comment:
Since the key is not used you can simply rename it to `_`

```suggestion
    for _, device_config in discovery_info.items():
```

Reply:
done